ml-agents/mlagents/trainers/tests/test_trainer_controller.py

```diff
@@ -283,14 +283,8 @@ def test_initialize_invalid_trainer_raises_exception(BrainInfoMock):
     tc = basic_trainer_controller(brain_info_mock)
     bad_config = dummy_bad_config()
 
-    try:
+    with pytest.raises(UnityEnvironmentException):
         tc.initialize_trainers(bad_config)
-        assert (
```

Comment:
mypy correctly complained that the assert was always true. `pytest.raises` is a cleaner way to test that an exception is raised